ninja_extra/apps.py

```diff
@@ -27,6 +27,7 @@ def ready(self) -> None:
         if app:  # pragma: no cover
             app.ready()
             self.injector = app.injector
+            self.injector.binder.install(self.ninja_extra_module)
```

Comment:
@QuentinRillet this has to be moved inside the `if` condition where we are checking for the presence of django_inector. That way we don't get to register NinjaModule twice.

Thanks for working on this